bin/list-all

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+GH_REPO="https://github.com/dojoengine/dojo"
+
+# Check for curl and git dependencies
+if ! command -v curl &>/dev/null; then
+    echo "Error: curl is not installed." >&2
+    exit 1
+fi
+
+if ! command -v git &>/dev/null; then
+    echo "Error: git is not installed." >&2
+    exit 1
+fi
+
+all_versions=$(git ls-remote --tags --refs "$GH_REPO" |
```

Comment:
Just seen that scarb also used `git`. Perhaps using GitHub API it should be only relying on `curl`? But as a contributor mentioned, the GitHub API is paginated. So a bit of logic implied. This one is not paginated @Jonatan-Chaverri?

Reply:
No, git command will return all the tags with no pagination.